content/fast/defer-unused-css/index.md

```diff
@@ -0,0 +1,150 @@
+<noscript><link rel="stylesheet" href="styles.css"></noscript>
+```
+
+This is not the standard way of loading CSS. Here's how it works:
+
+* `link rel="preload" as="style"` requests the stylesheet asynchronously. You can learn more about `preload` in the [Preload critical assets guide](https://web.dev/fast/preload-critical-assets).
+* The `onload` attribute in the `link` allows the CSS to be processed when it finishes loading.
+* "nulling" the `onload` handler once it is used helps some browsers avoid re-calling the handler upon switching the rel attribute.
+* The reference to the stylesheet inside of a `noscript` element works as a fallback for browsers that don’t execute JavaScript.
+
+The [resulting page](https://defer-css-optimized.glitch.me/) looks exactly like the previous version, even when most of its styles are loaded asynchronously:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    allow="geolocation; microphone; camera; midi; encrypted-media"
+    src="https://glitch.com/embed/#!/embed/defer-css-optimized?path=index.html&previewSize=100&attributionHidden=true"
+    alt="defer-css-optimized on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## Monitor
+
+Use the Devtools to run another **Performance** trace on the [optimized page](https://defer-css-optimized.glitch.me/). 
+
+The **FCP** marker appears before the page requests the CSS, which means the browser doesn’t need to wait for the CSS to load before rendering the page:
+
+<figure>
+  <img src="./cdt-perf-optimized.png"
+       alt="DevTools performance trace for unoptimized page, showing FCP starting before CSS loads." class="screenshot">
+</figure>
+
+As a final step, run Lighthouse on the optimized page. 
+
+In the report you’ll see that the FCP page has been reduced by **0.2s** (a 20% improvement!):
+
+<figure>
+  <img src="./lighthouse-metrics-optimized.png"
+       alt="Lighthouse report, showing an FCP value of '0.8s'." class="screenshot">
+</figure>
+
+Also, as a result, the **Eliminate render-blocking resources** suggestion is no longer under **Opportunities**, and now belongs to the **Passed Audits** section:
+
+<figure>
+  <img src="./lighthouse-opportunities-optimized.png"
+       alt="Lighthouse report, showing 'Eliminate blocing resources' on the 'Passed Audits' section." class="screenshot">
+</figure>
+
+## Next steps & references
+
+In this guide you used vanilla code to implement this optimization. In a real production scenario, it’s a good practice to use functions like [loadCSS](https://github.com/filamentgroup/loadCSS/blob/master/README.md), that can encapsulate this behavior and work well across browsers. As a complement to this, you can use tools like [Critical](https://github.com/addyosmani/critical/blob/master/README.md), which helps extracting and inlining the “Above the Fold” CSS you did manually in the **Optimize** step.
```

Comment:
General comment that I think it would be very cool to show developers how to use loadCSS and the Critical tool, as part of optimizing CSS story-- rather than pointing off to the generic resources.

Is this something that could be in a next guide? Or maybe a codelab? This is really my main comment-- I'd like to see these ideas explained more interactively.

Reply:
According to @addyosmani, there is a guide that @mihajlija is working on.
The idea was to add a reference to that guide at this part of the article, once is ready, instead to just the links to the tools.

Reply:
Perfect. That sounds great.